homeassistant/components/device_tracker/__init__.py

```diff
@@ -747,3 +756,32 @@ def get_gravatar_for_email(email: str):
     import hashlib
     url = 'https://www.gravatar.com/avatar/{}.jpg?s=80&d=wavatar'
     return url.format(hashlib.md5(email.encode('utf-8').lower()).hexdigest())
+
+
+def get_googlepluspthumb_from_email(email: str):
+    """Return a thumbnail image of a user's google plus profile image."""
+    import urllib.request
+    import urllib.parse
+    import urllib.error
+    from re import search
+
+    url = r'https://picasaweb.google.com/data/entry/api/user/'
+    try:
+        url += urllib.parse.quote(email) + r'?alt=json'
+        jsontext = urllib.request.urlopen(url).read().decode('utf-8')
+    except TypeError:
+        print(email)
```

Comment:
Don't use print. You can use LOGGER.error

Reply:
@danielhiverson does this satisfy your change request?